lib/core.ps1

```diff
@@ -1009,26 +964,6 @@ function get_shim_path() {
     return $shim_path
 }
 
-function search_in_path($target) {
```

Comment:
There are references of this fucntion in some manifests:

https://github.com/search?q=org%3AScoopInstaller+search_in_path+language%3AJSON&type=code&l=JSON
https://github.com/search?q=search_in_path+language%3AJSON&type=code&p=2

Reply:
I'll change that manifest, there's only one here.